xbmc/playlists/PlayListPLS.cpp

```diff
@@ -79,19 +79,18 @@ bool CPlayListPLS::Load(const std::string &strFile)
     return false;
   }
 
-  char szLine[4096];
   std::string strLine;
+  strLine.reserve(4096);
```

Comment:
imo all these "reserve" (also on other cases shown here) are not really needed
anyway keep a value of 4096 for most of time is too high, can be lowered to 1024